test/e2e/sourcery.js

```diff
@@ -41,8 +42,21 @@ module.exports = {
                 .waitForElementVisible('.user-info__title');
             }
         });
+        
+        //Assert if fields in menu are present
+        browser.assert.containsText('.main-nav', 'Time Logging');
+        browser.assert.containsText('.main-nav', 'Invoices');
+        browser.assert.containsText('.main-nav', 'Projects');
+        browser.assert.containsText('.main-nav', 'Clients');
+        browser.assert.containsText('.main-nav', 'Time Entries');
+        browser.assert.containsText('.main-nav', 'Tasks');
```

Comment:
Not mandatory but you could get rid of "browser" in every line of code so readability is a bit better.